### PR TITLE
Fix exception handling

### PR DESCRIPTION
**Pull request by chrwend**

### CODE REVIEW

File: Classes/ApplicationInsights.h [hunk 1]
```diff
@@ -14,7 +14,7 @@
 // Notification message which MSAIApplicationInsights is listening to, to retry requesting updated from the server
 #define MSAINetworkDidBecomeReachableNotification @"MSAINetworkDidBecomeReachable"
 
-#define MSAI_SERVER_URL   @"https://dc.services.visualstudio.com/v2/track"
+#define MSAI_SERVER_URL   @"https://eus-breeziest-in.cloudapp.net/v2/track"
```
Comment (lumaxis) [inline Classes/ApplicationInsights.h]: Why this change? The URLs should effectively point to the same server but I don't know if the latter one is guaranteed to be stable and stay the same.

**Comment by lumaxis** [comment]
Should I test this again? Code looks good.

**Comment by lumaxis** [comment]
Another quick question: Does this PR fix the `Time` field issue? Also, do we need the same fix in the OS X SDK?

**Comment by chrwend** [comment]
@lumaxis Yes, the time issue is also fixed by this. can you please merge
